packages/suite/src/views/wallet/transactions/components/TransactionSummary/components/SummaryCards/index.tsx

```diff
@@ -67,21 +68,23 @@ const SummaryCards = ({
                     </HiddenPlaceholder>
                 }
                 secondaryValue={
-                    <>
-                        <FormattedDate
-                            value={dataInterval[0] * 1000}
-                            year="numeric"
-                            month="short"
-                            day="2-digit"
-                        />{' '}
-                        -{' '}
-                        <FormattedDate
-                            value={dataInterval[1] * 1000}
-                            year="numeric"
-                            month="short"
-                            day="2-digit"
-                        />
-                    </>
+                    fromTimestamp && toTimestamp ? (
+                        <>
+                            <FormattedDate
+                                value={fromTimestamp * 1000}
+                                year="numeric"
+                                month="short"
+                                day="2-digit"
+                            />{' '}
+                            -{' '}
```

Comment:
I know that was there before but I get a twitch every time I see `{' '}`. I've been guilty of using it myself but at some point I'd like to go through these in the code base and replace them with `&nbsp;` where it makes sense.

Reply:
Creating empty strings increases the number of text nodes. It's not a huge deal but I generally favour `&nbsp;` (because it's easy to remember) or `&#32;` for a normal space. And in my very subjective opinion, `{' '}` is not aesthetically pleasing to ready in code. It feels more like a hack to me :slightly_smiling_face:

Reply:
you must be joking
https://stackoverflow.com/a/41142436